multi_camera_parallel_grabber/ex-multi_camera_parallel_grabber.cpp

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (C) 2019 ifm electronic, gmbh
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+// ex-multi_camera_parallel_grabber.cpp
+//
+// setup : refer README.md file
+//
+// Prerequisites:
+// *) one ifm 3D camera should be configured to use "Process Interface" for trigger.
+// *) All other camera must be in hardware trigger mode
+// *)
+//
+
+#include <iostream>
+#include <memory>
+#include <chrono>
+#include <vector>
+#include <thread>
+#include <atomic>
+#include <mutex>
+#include <signal.h>
+#include <array>
+#include <tuple>
+#include <ifm3d/camera.h>
+#include <ifm3d/fg.h>
+#include <ifm3d/image.h>
+#include <opencv2/core/core.hpp>
+#include <opencv2/imgcodecs.hpp>
+#include <opencv2/highgui/highgui.hpp>
+
```

Comment:
Code doesn't compile on Ubuntu 18.04. Missing `#include <iomanip> for `std::setfill`, `std::put_time`, and `std::setw`